### DISCUSSION TITLE
tmux-setup but no highlight

### DISCUSSION THREAD

**Discussion by wangrongwei** [hugsy/gef#1143] [Q&A] [answered]
gdb and gef version:
gdb:9.2
gdb-python:3.8

When I run tmux-setup, it can split a single window in right, but no any highlight, likes:
![image](https://github.com/user-attachments/assets/c740cdca-b92a-4bff-87e1-e2c68b8b80ef)

I'm not sure whether need to update the gef. And I can sure my tmux and gef has below configure:
set -g default-terminal xterm-256color
gef config gef.disable_color 1

Thanks for any suggestions!

**Comment by hugsy** [comment, marked as answer]
`gef config gef.disable_color 1` means the colors **are** disabled.

Try `gef config gef.disable_color 0` to undo that setting, followed by `gef save` if you want to make it permanent.

A good way to go about this kind of issue is to isolate the root cause: you can try reseting gef's settings, by (re)moving ~/.gef.rc and restart gef. If you still don't have colors (default in gef), it must come from your tmux. 

Cheers

**Reply by wangrongwei** [reply]
I have found the root cause is debug=True

Thanks for your reminder.

**Reply by hugsy** [reply]
Weird, `gef.debug` doesn't disable colors. It just increases the amount of logging info displayed by GEF.

Regardless, glad it was solved 🎉

**Reply by skirge** [reply]
same for me, no colors in tmux with gef.debug = 1

**Comment by wangrongwei** [comment]
Hi, hugsy

Thanks, I configure gef.disable_color=false, it can't work, but it works when I remove ~/.gef.rc. This is my .gef.rc

[gef.rc.tar.gz](https://github.com/user-attachments/files/17221196/gef.rc.tar.gz)

can you help me check which options lead to it? 

Anyway, thanks.

**Reply by hugsy** [reply]
Not sure if related, but there's no need for the `=` in the syntax, just :

```
gef config gef.disable_color false
```
